libs/datasets/combined_datasets.py

```diff
@@ -21,18 +22,19 @@
 from libs.datasets import data_source
 from libs.datasets import dataset_pointer
 from libs.datasets import manual_filter
-from libs.datasets.custom_aggregations import ALL_NYC_REGIONS
 from libs.datasets.dataset_pointer import DatasetPointer
 from libs.datasets.dataset_utils import DatasetType
 from libs.datasets.sources.hhs_hospital_dataset import HHSHospitalStateDataset
 from libs.datasets.sources.hhs_hospital_dataset import HHSHospitalCountyDataset
 from libs.datasets.sources.test_and_trace import TestAndTraceData
 from libs.datasets.timeseries import MultiRegionDataset
 from libs.datasets.timeseries import OneRegionTimeseriesDataset
-from libs.datasets.sources.nytimes_dataset import NYTimesDataset
+from libs.datasets.sources.cdc_nyt_combined_cases_deaths import (
+    CdcNytCombinedCasesDeaths,
```

Comment:
Yeah, our naming is kinda all over the place for these datasets 😞 

My weakly-held thoughts: 
- I think it's helpful to note what variables are in this dataset because we have a different CDC dataset for vaccinations, and it could become unclear what dataset contains what. 
- Since this contains both state and county data, I feel it might be unnecessary to include the region levels in the name (but I could be convinced otherwise!). 
- We typically don't distinguish sub-sources from within the CDC (e.g. the CDC Covid Data Tracker vs a published CDC dataset), although I think there's an argument that we should be. 

I think I'd be inclined to merge this with the name as-is (`CdcNytCombinedCasesDeaths`), but am happy to modify/change the name in a follow-up PR if you have strong thoughts